package.json

```diff
@@ -50,19 +52,9 @@
     "eslint-plugin-babel": "^4.1.2",
     "eslint-plugin-import": "^2.16.0",
     "istanbul-instrumenter-loader": "^3.0.1",
-    "jasmine-core": "^2.99.1",
-    "karma": "^3.0.0",
-    "karma-babel-preprocessor": "^7.0.0",
-    "karma-browserify": "^5.3.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage": "^1.1.2",
-    "karma-fixture": "^0.2.6",
-    "karma-html2js-preprocessor": "^1.1.0",
-    "karma-jasmine": "^1.1.2",
-    "karma-spec-reporter": "0.0.31",
-    "karma-threshold-reporter": "^0.1.15",
     "npm-run-all": "^3.1.2",
     "onchange": "^3.2.1",
-    "prismjs": "^1.8.1"
+    "prismjs": "^1.8.1",
+    "jest": "^23.6.0"
```

Comment:
Acho que a versão mais atual é a 24.1.0

Reply:
I've tried to update Jest to it's latest release and remove babel-jest, since it is already included in jest at this version, but the babel-jest included asks for Babel 7 and the project doesn't use it yet :/

Reply:
Oh ok 👍